packages/preact-query/src/__tests__/useMutationState.test-d.tsx

```
@@ -1,4 +1,8 @@
-import type { MutationState, MutationStatus } from '@tanstack/query-core'
+import type {
+  Mutation,
+  MutationState,
+  MutationStatus,
+} from '@tanstack/query-core'
```

ESLint

[error] `@tanstack/query-core` type import should occur after import of `../useMutationState`

(import/order)